DeliBuddy/data/src/main/java/yapp/android1/data/entity/AddressMapper.kt

```diff
@@ -0,0 +1,29 @@
+package yapp.android1.data.entity
+
+import yapp.android1.domain.entity.Address
+
+object AddressMapper {
+    fun responseToAddress(keywordDocument: KeywordDocument): Address {
```

Comment:
```suggestion
    fun toAddress(keywordDocument: KeywordDocument): Address {
```
이 정도로도 직관적이라 생각해서 제안드려용 ㅎㅎ